src/pudl/output/pudltabl.py

```diff
@@ -116,15 +116,14 @@ def __init__(
 
         # grab all working eia dates to use to set start and end dates if they
         # are not set
-        eia_dates = pudl.helpers.get_working_eia_dates()
         if start_date is None:
-            self.start_date = min(eia_dates)
+            self.start_date = min(pudl.helpers.get_working_dates_by_datasource("ferc1"))
         else:
             # Make sure it's a date... and not a string.
             self.start_date = pd.to_datetime(start_date)
 
         if end_date is None:
-            self.end_date = max(eia_dates)
+            self.end_date = max(pudl.helpers.get_working_dates_by_datasource("eia"))
```

Comment:
I believe most of the `pudl_out` settings in the tests do not have start/end dates. the date ranges in the tests are mostly a factor of how much data is in a database which you've run the tests on. These default ranges basically ensure that no matter what your db contains, you'll get all of the data out. Most of the date selections happening in the eia outputs assume you can have a `None` start/end date, which I believe pre-dates us adding this default date value, but tbh i'm not sure. If I didn't know we were about to rip a lot of the current `PudlTabl` out, I would probably have gone in and standardized up the way in which the `report_date` table selections were happening as well.

And yes the ferc1 output tests are lacking at best. there are much more actual validation happening in the validation tests (lol hence the name). These ferc1 output tables are truly the canonical examples of tables that should really be views. Not a whole lot is happening in any of these functions beyond read/merge/maybe rename with the exception of the addition of a few very basic calculated columns. But it would definitely be wise to take a step back and ask what types of tests should be happening on these tables. I think with the migration to using views we'll need to define table schemas and such, which would be a very good time imo to enforce our expectations on these lil guys.